homeassistant/components/youtube/strings.json

```diff
@@ -0,0 +1,42 @@
+{
+  "config": {
+    "abort": {
+      "access_not_configured": "Please read the below message we got from Google:\n\n{message}",
+      "already_configured": "[%key:common::config_flow::abort::already_configured_account%]",
+      "reauth_successful": "[%key:common::config_flow::abort::reauth_successful%]",
+      "unknown": "[%key:common::config_flow::error::unknown%]"
+    },
+    "error": {
+      "invalid_auth": "[%key:common::config_flow::error::invalid_auth%]",
+      "cannot_connect": "[%key:common::config_flow::error::cannot_connect%]"
+    },
+    "step": {
+      "channels": {
+        "description": "Select the channels you want to add.",
+        "data": {
+          "channels": "YouTube channels"
+        }
+      }
+    }
+  },
+  "options": {
```

Comment:
There's no options flow. Please remove strings that aren't used at the moment.

Reply:
Oh whoops. I have #93667 for that. Do you want me to create a PR to revert this to add it 1 PR later?

Reply:
It's best to address the small comments in a single PR that I can merge immediately. Larger work may take longer time and I won't personally prioritize anything that I haven't commented on.